src/main/java/Duke.java

```diff
@@ -1,10 +1,176 @@
+import java.io.File;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+import java.io.PrintWriter;
+import java.io.IOException;
+
 public class Duke {
-    public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+
+    final static String ENTRY = "Hello! I'm Duke\nWhat can I do for you?";
+    final static String EXIT_COMMAND = "bye";
+    final static String LIST_COMMAND = "list";
+    final static String DELETE_COMMAND = "delete";
+    final static String MARK_COMMAND = "mark";
+    final static String UNMARK_COMMAND = "unmark";
+    final static String TODO_COMMAND = "todo";
+    final static String DEADLINE_COMMAND = "deadline";
+    final static String EVENT_COMMAND = "event";
+    final static DateTimeFormatter inputFormatter = DateTimeFormatter.ofPattern("yyyy/MM/d HHmm");
+
+    final static DateTimeFormatter outputFormatter = DateTimeFormatter.ISO_LOCAL_DATE_TIME;
+
+
+    public static ArrayList<Task> list;
+
+    public static void main(String[] args) throws IOException {
+        System.out.println(ENTRY);
+        Scanner sc = new Scanner(System.in);
+        list = new ArrayList<Task>();
+        while (true) {
```

Comment:
Perhaps true can be saved as variable so that it is clear under what conditions this loop will iterate.

Reply:
Resolved.